layouts/partials/inline/inline.css

```diff
@@ -0,0 +1,219 @@
+'}' +
+
+'.iframe-content .tvp-videos-container .tvp-videos-arrows > svg {' +
+	'fill: {{- ($.Scratch.Get "settings").videos_carousel_arrow_color -}};' +
+'}' +
+
+'.iframe-content .tvp-videos-scroller .tvp-video .tvp-video-title{' +
+	'color: {{- ($.Scratch.Get "settings").videos_carousel_item_title_font_color -}};' +
+	'font-size: calc({{- ($.Scratch.Get "settings").videos_carousel_item_title_font_size_min -}}px + ({{- ($.Scratch.Get "settings").videos_carousel_item_title_font_size_max }} - {{ ($.Scratch.Get "settings").videos_carousel_item_title_font_size_min -}}) * ((100vw - 300px) / (1600 - 300)));' +
+	'font-weight: {{- ($.Scratch.Get "settings").videos_carousel_item_title_font_weight -}};' +
+	'width: {{- ($.Scratch.Get "settings").videos_carousel_item_title_width -}};' +
+	'display: {{- ($.Scratch.Get "settings").videos_carousel_item_title_display -}};' +
+	'padding: {{- ($.Scratch.Get "settings").videos_carousel_item_title_padding -}};' +
+	'position: {{- ($.Scratch.Get "settings").videos_carousel_item_title_position -}};' +	
+	'top: {{- ($.Scratch.Get "settings").videos_carousel_item_title_top -}};' +	
+	'opacity: {{- ($.Scratch.Get "settings").videos_carousel_item_title_inactive -}};'+
+	'-webkit-transform: {{- ($.Scratch.Get "settings").videos_carousel_item_title_transform -}};' +	
+	'transform: {{- ($.Scratch.Get "settings").videos_carousel_item_title_transform -}};' +
+'}' +
+
+'.iframe-content .tvp-videos-scroller .tvp-video-active .tvp-video-title ' +
+', .iframe-content .tvp-videos-scroller .tvp-video:hover .tvp-video-title{' +
+	'opacity: {{- ($.Scratch.Get "settings").videos_carousel_item_title_active -}};'+
+'}' +
+
+'.iframe-content .tvp-videos-scroller .tvp-video{' +
+	'border: {{- ($.Scratch.Get "settings").videos_carousel_item_border -}};' +	
+'}' +
+
+'.iframe-content .tvp-video-active .tvp-video-image-overlay' +
+', .iframe-content .tvp-videos-scroller .tvp-video:hover .tvp-video-image-overlay{' +
+	'opacity: {{- ($.Scratch.Get "settings").videos_carousel_item_thumbnail_active -}};'+
+'}' +
+
+'.iframe-content .tvp-video-image-overlay{' +
+	'background-color: {{- ($.Scratch.Get "settings").videos_carousel_item_thumbnail_overlay_color -}};'+
+	'opacity: {{- ($.Scratch.Get "settings").videos_carousel_item_thumbnail_inactive -}};'+
+'}' +
+
+'.iframe-content .tvp-videos-scroller .tvp-video:before{' +
+	'padding-top: {{- ($.Scratch.Get "settings").videos_carousel_item_height -}};' +
+'}' +
+
+'.iframe-content .tvp-videos-scroller .tvp-video .tvp-video-image:before{' +
+	'padding-top: {{- ($.Scratch.Get "settings").videos_carousel_item_thumbnail_height -}};' +	
+'}' +
+
+//all media queries here
+'@media only screen and (max-width: 769px){' +
+	'.iframe-content .tvp-row-titles h2, .iframe-content .tvp-row .tvp-mobile-product-title h2{' +
+		'line-height: 14px;' +
+	'}' +
+
+	'.iframe-content .tvp-products-scroller #productContent .tvp-product-item .tvp-product-info-view-details{' +
```

Comment:
Why do we need such long CSS selectors?
Can't we just leave `.tvp-product-info-view-details {` ?